src/main/resources/view/DarkTheme.css

```
@@ -132,6 +132,13 @@
     -fx-text-fill: #010504;
 }
 
+.cell_header_label {
+    -fx-font-family: "Segoe UI Semibold";
+    -fx-font-size: 30px;
+    -fx-padding: 10 0 10 0;
```

I feel that the label is a little too big.

```suggestion
    -fx-font-size: 22px;
    -fx-padding: 0 0 10 0;
```